File: java/Program/Schedule.java
public class Schedule {
    private final String id;
    private int dayOfWeek;
    private String startTime;
    private String endTime;

    // ===== Constructor =====
    public Schedule(String id, int dayOfWeek, String startTime, String endTime) {
        this.id = id;
        this.dayOfWeek = dayOfWeek;
        this.startTime = startTime;
        this.endTime = endTime;
    }

    // ===== Getter =====
    public String getId() {
        return id;
    }

    public int getDayOfWeek() {
        return dayOfWeek;
    }

    public String getStartTime() {
        return startTime;
    }

    public String getEndTime() {
        return endTime;
    }

    // ===== Setter =====
    public void setDayOfWeek(int dayOfWeek) {
        this.dayOfWeek = dayOfWeek;
    }

    public void setStartTime(String startTime) {
        this.startTime = startTime;
    }

    public void setEndTime(String endTime) {
        this.endTime = endTime;
    }

    // ===== toString =====
    @Override
    public String toString() {
        String[] dayMap = {"Mon", "Tue", "Wed", "Thu", "Fri", "Sat", "Sun"};
        String d = (dayOfWeek >= 0 && dayOfWeek <= 6) ? dayMap[dayOfWeek] : String.valueOf(dayOfWeek);
        return "Schedule[ID=" + id + ", " + d + " " + startTime + "-" + endTime + "]";
    }
}

File: python/Program/Schedule.py
class Schedule:

    def __init__(self, id: str, day_of_week: int, start_time: str, end_time: str) -> None:

        self.__id = id
        self.__day_of_week = day_of_week
        self.__start_time = start_time
        self.__end_time = end_time

    # Getter untuk mengambil nilai dari masing-masing atribut
    def get_id(self) -> str:
        return self.__id

    def get_day_of_week(self) -> int:
        return self.__day_of_week

    def get_start_time(self) -> str:
        return self.__start_time

    def get_end_time(self) -> str:
        return self.__end_time

    # Setter untuk menyimpan nilai masing-masing atribut
    def set_day_of_week(self, day_of_week: int) -> None:
        self.__day_of_week = day_of_week

    def set_start_time(self, start_time: str) -> None:
        self.__start_time = start_time

    def set_end_time(self, end_time: str) -> None:
        self.__end_time = end_time

    def __str__(self) -> str:
        day_map = ["Mon", "Tue", "Wed", "Thu", "Fri", "Sat", "Sun"]
        d = day_map[self.__day_of_week] if 0 <= self.__day_of_week <= 6 else str(self.__day_of_week)
        return f"Schedule[ID={self.__id}, {d} {self.__start_time}-{self.__end_time}]"